main.go

```diff
@@ -24,7 +24,7 @@ func main() {
 		cli.StringFlag{
 			Name:   "webhook",
 			Usage:  "slack webhook url",
-			EnvVar: "SLACK_WEBHOOK",
+			EnvVar: "SLACK_WEBHOOK,PLUGIN_WEBHOOK",
```

Comment:
👍 What time do you merge this PR and update docker image?

Nice Job.

We need `environment` section to provide webhook url In `0.5` version now.

```
pipeline:
  slack:
    channel: drone
    username: drone
    environment:
      - SLACK_WEBHOOK=https://hooks.slack.com/services/xxxxxx